judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/scoreboard/ContestScoreboardService.java

```diff
@@ -22,4 +23,10 @@
             @PathParam("contestJid") String contestJid,
             @QueryParam("frozen") boolean frozen,
             @QueryParam("showClosedProblems") boolean showClosedProblems);
+
+    @POST
+    @Path("/refresh")
+    void refreshScoreboard(
+            @HeaderParam(AUTHORIZATION) Optional<AuthHeader> authHeader,
```

Comment:
```suggestion
            @HeaderParam(AUTHORIZATION) AuthHeader authHeader,
```

Because guests cannot hit this endpoint anyway